src/map/map.ts

```diff
@@ -659,9 +665,20 @@ export class PropertiesMap {
         // ======= color axis settings
         if (this._options.hasColors()) {
             // setup initial color range (must do before setting up events)
-            const values = this._colors(0)[0] as number[];
-            const { min, max } = arrayMaxMin(values);
+            const determineColorRange = (optionMin: number, optionMax: number) => {
```

Comment:
I could see a case where min/max is undefined, if the user only gives us one of the two. We should handle it

Reply:
If user didn't set it, the value is zero (due to the initialization of MapOptions):
```
this.color = {
    ...
    min: new HTMLOption('number', 0),
    max: new HTMLOption('number', 0),
};
```

Reply:
Right, I missed that. It's a bit unfortunate that we can not distinguish between "the user gave `min=-3` and not max" vs "the user gave `min=-3` and `max=0`". In the first case, we should compute the max from the array.

Should we change the default to `min: new HTMLOption('number', NaN)`, and check with `isnan` here? Using `NaN` like `undefined`.

Reply:
Indeed, good idea! Let me change it!